ota-plus-web/app/reactapp/src/containers/Updates.jsx

```diff
@@ -4,19 +4,32 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { observable } from 'mobx';
 import { observer, inject } from 'mobx-react';
+import { Pagination } from 'antd';
+
 import { UpdateCreateModal, UpdateHeader, UpdateList } from '../components/updates';
 import { Loader } from '../partials';
 
+import { UPDATES_LIMIT_PER_PAGE } from '../config';
+
+const PAGE_NUMBER_DEFAULT = 1;
+
 @inject('stores')
 @observer
 class Updates extends Component {
   @observable
   createModalShown = false;
+
   @observable
   updateDetailsShown = false;
+
   @observable
   selectedUpdate = null;
 
+  constructor(props) {
+    super(props);
+    this.state = { pageNumber: PAGE_NUMBER_DEFAULT };
```

Comment:
Any particular reason to have `state` declared in `constructor` in this case over shorthand equivalent?
```suggestion
    state = {
      pageNumber: PAGE_NUMBER_DEFAULT
    };
```

Reply:
Please see: https://reactjs.org/docs/state-and-lifecycle.html